gammapy/datasets/tests/test_map.py

```diff
@@ -746,6 +746,31 @@ def test_map_dataset_fits_io(tmp_path, sky_model, geom, geom_etrue):
     assert_allclose(stacked1.edisp.edisp_map, stacked.edisp.edisp_map)
 
 
+@requires_data()
+def test_map_dataset_fits_creation_metadata(tmp_path, sky_model, geom, geom_etrue):
+    dataset = get_map_dataset(geom, geom_etrue)
+
+    bkg_model = FoVBackgroundModel(dataset_name=dataset.name)
+    dataset.models = [sky_model, bkg_model]
+
+    dataset.meta.creation.creator = "MySoftware"
+    dataset.meta.creation.origin = "MyOrganization"
+
+    dataset.counts = dataset.npred()
+    dataset.mask_safe = dataset.mask_fit
+    gti = GTI.create([0 * u.s], [1 * u.h], reference_time="2010-01-01T00:00:00")
+    dataset.gti = gti
+
+    dataset.write(tmp_path / "test.fits")
+
+    hdul = fits.open(tmp_path / "test.fits")
+    for hdu in hdul:
+        assert "CREATOR" in hdu.header
+        assert "CREATED" in hdu.header
+        assert hdu.header["CREATOR"] == "MySoftware"
+        assert hdu.header["ORIGIN"] == "MyOrganization"
+
```

Comment:
Should we check also the "DATE", by fixing an arbitrary one?

Reply:
The date is updated to `now()` when writing.